pkg/apiserver/controllers/decisions.go

```diff
@@ -171,7 +171,7 @@ func (c *Controller) StreamDecision(gctx *gin.Context) {
 	}
 
 	// getting expired decisions
-	data, err = c.DBClient.QueryExpiredDecisionsSince(blockerInfo.LastPull.Add(-5 * time.Minute)) // do we want to give exactly lastPull time ?
+	data, err = c.DBClient.QueryExpiredDecisionsSince(blockerInfo.LastPull) // do we want to give exactly lastPull time ?
```

Comment:
I think you want to add at least one second to avoid race conditions ? (as it should be ok to give twice a deleted decision, but a drama if we missed one)